var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -13,18 +13,33 @@ class Fms(CMakePackage):
     system models."""
 
     homepage = "https://github.com/NOAA-GFDL/FMS"
-    url      = "https://github.com/NOAA-GFDL/FMS/archive/refs/tags/2021.02.01.tar.gz"
+    url      = "https://github.com/NOAA-GFDL/FMS/archive/refs/tags/2021.04.tar.gz"
+    git      = "https://github.com/NOAA-GFDL/FMS.git"
 
     maintainers = ['kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett']
 
     variant('64bit', default=True, description='64 bit?')
     variant('gfs_phys', default=True, description='Use GFS Physics?')
     variant('openmp', default=True, description='Use OpenMP?')
+    variant('enable_quad_precision', default=True, description='quad precision reals?')
+    variant('with_yaml', default=True, description='yaml input file support(requires libyaml)?')
```

Comment:
Can we avoid `with_*` or `enable_*`? These should be implied by `+` in the variant:
```suggestion
    variant('quad_precision', default=True, description='quad precision reals')
    variant('yaml', default=True, description='yaml input file support')
```
Also, we shouldn't have any question marks in description (users looking at Spack info might be confused by reading `(requires libyaml)?`)

Reply:
@alalazo It's updated now with those changes